### ISSUE TITLE
Django 1.11 compatibility issue - `cannot import name flatatt`

### ISSUE THREAD

**Issue by ses4j** [carltongibson/django-filter#673]
```File "/.../environments/mspro/lib/python2.7/site-packages/django_filters/widgets.py", line 13, in <module>
    from django.forms.widgets import flatatt
ImportError: cannot import name flatatt```

Got the above when trying to upgrade to Django 1.11.

**Comment by charlex** [comment]
Seeing this as well:

```
  File "/Users/cbodkin/.virtualenvs/p2p-liveblog/lib/python2.7/site-packages/django_filters/widgets.py", line 13, in <module>
    from django.forms.widgets import flatatt
ImportError: cannot import name flatatt
```

**Comment by charlex** [comment]
It looks like the proper way to import that is:

```
from django.forms.utils import flatatt
```

See:
https://docs.djangoproject.com/en/1.10/_modules/django/forms/widgets/

**Comment by rpkilby** [comment]
What version of django-filter are you using. This should have been fixed in the latest release (1.0.2)

**Comment by charlex** [comment]
Ah, yes. That fixed it for me. Thanks.
